Makefile

```diff
@@ -205,20 +204,14 @@ entrypoint.sh:
 #	make fluent.conf [DOCKERFILE=] [VERSION=]
 
 fluent.conf:
-	mkdir -p docker-image/$(DOCKERFILE)/conf
-	docker run --rm -i -v $(PWD)/templates/conf/fluent.conf.erb:/fluent.conf.erb:ro \
-		ruby:alpine erb -U -T 1 \
-			dockerfile='$(DOCKERFILE)' \
-			version='$(VERSION)' \
-		/fluent.conf.erb > docker-image/$(DOCKERFILE)/conf/fluent.conf
+	make container-image-template TEMPLATE_FILE=conf/fluent.conf
 
 # Generate kubernetes.conf from template.
 #
 # Usage:
 #	make kubernetes.conf [DOCKERFILE=] [VERSION=]
 
 kubernetes.conf:
-	mkdir -p docker-image/$(DOCKERFILE)/conf
```

Comment:
Memo: Just removing this line in this commit isn't appropriate, but it's cared in the following commit.

Reply:
Oops, rebase stuff...